heareval/tasks/pipeline.py

```diff
@@ -261,6 +286,18 @@ def run(self):
                 "%s embedding_type unknown" % self.data_config["embedding_type"]
             )
 
+        # Remove files from the process metadata which donot exist in the dataset.
```

Comment:
spelling

Reply:
Is there a way to know if small mode is enabled? In which case we can run this only when small is enabled.